### ISSUE TITLE
Replace luabase API with forta labels API

### ISSUE THREAD

**Issue by mwakaba2** [NethermindEth/general-agents-module#55]
@RCantu92 Luabase is planning to sunset the query API in early February, so the victim identification bot will not be able to access ethereum, fantom, and polygon tags. To avoid any disruption, we created a new API to replace Luabase.  

You can now query the tags with the following endpoints:
```
https://api.forta.network/labels/state?entities=<ADDRESS>&sourceIds=<LABELS_SOURCE>
```
For ethereum tags --> set `sourceIds=etherscan-tags`
For polygon tags --> set `sourceIds=polygon-tags`
For fantom tags --> set `sourceIds=fantom-tags`

Here's an example getting an ethereum tag for the address `0xe74b28c2eAe8679e3cCc3a94d5d0dE83CCB84705`.
You can get the tag from `events[idx]["label"]["label"]`
```bash
$ curl "https://api.forta.network/labels/state?entities=0xe74b28c2eAe8679e3cCc3a94d5d0dE83CCB84705&sourceIds=etherscan-tags" | jq
{
  "events": [
    {
      "id": "0xb1f5af8d8f36fe96844974f0bfe2deed264010b8975aa37e84f37f5e990f1333",
      "created_at": "2023-01-11T19:43:03.015925716Z",
      "label": {
        "label": "Wintermute Exploiter",
        "confidence": 1,
        "entity": "0xe74b28c2eae8679e3ccc3a94d5d0de83ccb84705",
        "entityType": "ADDRESS",
        "remove": false
      },
      "source": {
        "id": "etherscan-tags"
      }
    }
  ]
}
```
You can also pass in multiple addresses. They just need to be comma-separated:
```
 curl "https://api.forta.network/labels/state?entities=0xe74b28c2eAe8679e3cCc3a94d5d0dE83CCB84705,0x000002Cba8DfB0a86A47a415592835E17fac080a&sourceIds=etherscan-tags" | jq
```
However, if you only expect 1 tag for 1 address, adding this parameter will make the API response faster: `&limit=1`.

```typescript
const fetchLuabaseDb = async (address: string, chain: string): Promise<string> => {
  if (restApis["luabaseKey"] === "") return "";

  const sqlQuery: string = `
        select tag
        from ${chain}.tags
        where address='${address}'
        limit 15
      `;
```
For a smooth transition, could you replace the above [Luabase endpoint/code](https://github.com/NethermindEth/general-agents-module/blob/main/src/utils/victim-identification/victim.identifier.ts#L117) with this new labels API before February 1st?

**Comment by Vxatz** [comment]
Thank you for bringing this to our attention @mwakaba2, we'll work on it. 👍

**Comment by Vxatz** [comment]
Fixed in #56.
